canonical-web-and-design/jaas.ai
# Remove unused files

## Done

- Removed JS, CSS and images that were not being used.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Have a poke around, everything should be unchanged.

## Details

- Should help complete #188.
